File: 2020/02/04/cube.js
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int [] returnValue = {0,0};
        if(nums != null){
            int firstValue=0;
            int secondValue=0;
            
            for(int i = 0; i < nums.length; i++ ){
                firstValue = nums[i];
                for(int k = i+1; k < nums.length; k++ ){
                    secondValue = nums[k];
                    if(firstValue + secondValue == target){
                        returnValue[0] = i;
                        returnValue[1] = k;
                    }
                }
            }
        }
        return returnValue;
    }
}

File: 2020/03/02/cube.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int [] result = new int[nums.length];
        for(int i = 0; i < nums.length; i++){
            int cnt = 0;
            for(int k = 0; k < nums.length; k++){
                if(nums[i] > nums[k]){
                    cnt++;
                }
                result[i] = cnt;
            }
        }
        return result;
    }
}
